v1.0.7 Old CSS files will be stored up to one day

timgblack opened the pull request.

Commit:
v1.0.7 Old CSS files will be stored up to one day

thejamescollins commented:
This looks good @timgblack.

While you're at it, I think we should also add PHPdoc comments to each function, particularly the new ones and the ones that changed  as part of  3712777.

Thanks.

Commit:
Added PHPDoc comments

thejamescollins committed:
Avoid PHP warning when cleaning up old CSS files for the first time

PHP Warning:  Invalid argument supplied for foreach() in wp-content/plugins/om4-custom-css/om4-custom-css.php on line 331

This was occurring because the om4_freeform_css_old_files option was returning false (instead of an empty array) the very first time it was loaded after upgrading from v1.0.6 to v1.0.7

thejamescollins committed:
Remove redundant false, and improved PHPDoc

thejamescollins committed:
Readme changelog update for v1.0.7

thejamescollins committed:
Explain use of delete_option() & add_option()
